site/app/Resources/views/template/email_notification_template.html.twig

```diff
@@ -0,0 +1,253 @@
+        text-align: center;
+      }
+
+      td[class="no-pad"]{
+        padding: 0 0 20px 0 !important;
+      }
+
+      td[class="no-padding"]{
+        padding: 0 !important;
+      }
+
+      td[class="section-padding"]{
+        padding: 50px 15px 50px 15px !important;
+      }
+
+      td[class="section-padding-bottom-image"]{
+        padding: 50px 15px 0 15px !important;
+      }
+
+      /* ADJUST BUTTONS ON MOBILE */
+      td[class="mobile-wrapper"]{
+        padding: 10px 5% 15px 5% !important;
+      }
+
+      table[class="mobile-button-container"]{
+        margin:0 auto;
+        width:100% !important;
+      }
+
+      a[class="mobile-button"]{
+        width:80% !important;
+        padding: 15px !important;
+        border: 0 !important;
+        font-size: 16px !important;
+      }
+
+    }
+  </style>
+</head>
+<body style="margin: 0; padding: 0;">
+
+<!-- HEADER -->
+<table border="0" cellpadding="0" cellspacing="0" width="100%" style="table-layout: fixed;">
+  <tr>
+    <td bgcolor="#ffffff">
+      <div align="center" style="padding: 0px 15px 0px 15px;">
+        <table border="0" cellpadding="0" cellspacing="0" width="500" class="wrapper">
+          <!-- LOGO/PREHEADER TEXT -->
+          <tr>
+            <td style="padding: 20px 0px 30px 0px;" class="logo">
+              <table border="0" cellpadding="0" cellspacing="0" width="100%">
+                <tr>
+                  <td bgcolor="#fff" width="100" align="center">
+                    <a href="https://www.librecores.org" target="_blank">
+                      <img src="{{ logoUrl }}"
+                           class="img-responsive" style="display: block;
+                           font-family: Helvetica, Arial, sans-serif; color: #666666; font-size: 16px;" border="0">
+                    </a></td>
+                </tr>
+              </table>
+            </td>
+          </tr>
+        </table>
+      </div>
+    </td>
+  </tr>
+</table>
+
+<!-- ONE COLUMN SECTION -->
+<table border="0" cellpadding="0" cellspacing="0" width="100%" style="table-layout: fixed;">
+  <tr>
+    <td bgcolor="#ffffff" align="center" class="section-padding">
+      <table border="0" cellpadding="0" cellspacing="0" width="500" class="responsive-table">
+        <tr>
+          <td>
+            <table width="100%" border="0" cellspacing="0" cellpadding="0">
+              <tr>
+                <td>
+                  <!-- COPY -->
+                  <table width="100%" border="0" cellspacing="0" cellpadding="0">
+                    <tr>
+                      <td align="left" style="font-size: 25px; font-family: Helvetica, Arial, sans-serif; color: rgba(63, 176, 172, 1); padding-top: 30px;" class="padding-copy">Hi {{ username }},</td>
+                    </tr>
+                    <tr>
+                      <td align="left" style="font-size:14px;padding: 20px 0 5px 0;border-bottom: 2px solid #e2e2e2">You have a new Notification from <b>LibreCores</b>.</td>
```

Comment:
Notification -> notification

And it's easier to read if you limit the line to 100 characters.